tutorials/text/usetex.py

```diff
@@ -51,6 +51,32 @@
       "font.family": "serif",
       "font.serif": ["Palatino"],
   })
+  # It's also possible to use the reduced notation by directly setting font.family:
+  plt.rcParams.update({
+    "text.usetex": True,
+    "font.family": "Helvetica"
+  })
+
+Currently, the supported fonts are::
+
+    +---------------------+--------------------------------------------------+
+    | family              | fonts                                            |
+    +=====================+==================================================+
+    | ``'serif'``         | ``'New Century Schoolbook'``, ``'Bookman'``,     |
```

Comment:
The backtick is one space earlier here than in the next rows. I'm not sure if that will make a difference since the table isn't rendered yet.